nifi-nar-bundles/nifi-smb-bundle/nifi-smb-nar/pom.xml

```diff
@@ -35,5 +35,16 @@
             <artifactId>nifi-smb-processors</artifactId>
             <version>1.17.0-SNAPSHOT</version>
         </dependency>
+        <!--dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-smb-smbj-client</artifactId>
+            <version>1.17.0-SNAPSHOT</version>
+        </dependency-->
```

Comment:
I think we don't want to merge it this way. Can we remove it instead of just commenting out?